### ISSUE TITLE
How do I parse what should be the graph-level logits?

### ISSUE THREAD

**Issue by cinjon** [rusty1s/pytorch_geometric#2055]
## ❓ Questions & Help

I have a graph and I am running it through a couple of GCNConvs in order to have my policy take an action on that graph. In order to do this, I am producing, for each batch, a ```Batch.from_data_list([Data(edge_index=torch.tensor(<...>), x=torch.tensor(<...>))])```. In other words, I create a list of geometric.data.Data graphs and feed that to geometric.data.Batch in order to coalesce it into a single entity to go through the network.

On the other side of this, I get logits that are of the size ```[Total number of nodes in all entries in the batch, num_output]```. Consequently it appears that these are logits for _every_ node in each graph whereas my ideal would be a single set of logits for the entire graph. 

What is the common approach at this point in the GNN literature in order to get a graph level result? Is it common to now aggregate the logits for each graph? Or is there another approach that's better suited for this structure?

Thanks.

**Comment by cinjon** [comment]
Ok, I found this: https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.glob.global_mean_pool. Seems like these pooling approaches are still the best approach.
